### ISSUE TITLE
about the error of crf

### ISSUE THREAD

**Issue by OKtang** [UmasouTTT/keras_bert_ner#1]
hi,
when I run the crf layer from the keras-contrib, I always got an error.  So I want to konw what's the right  edition of Keras-contrib and keras.  And do you konw the other crf layer which can be used in tensorflow? Thank you very much!
best.

**Comment by UmasouTTT** [comment]
Keras version:2.2.4
keras-contrib version:2.0.8
Sorry, I am not sure if there are other packaged libraries for crf.
I updated the project source code so that the bert layer can also be trained.
By the way ，it's a good choice to use other training data, because I have been unable to upload large resource files to git.

**Comment by OKtang** [comment]
Thank you for your reply! and did you evaluate your results ,how about it? When I predict on my test dataset, the  label is almost reversed. Do you have some suggestion for me. Thanks!

**Comment by UmasouTTT** [comment]
This model can achieve more than 98 percent accuracy on a generic dataset. But the version I released at the beginning does have some problems. These issues have now been fixed. Please download the latest version and try it out. If there are any problems, please let me know~

**Comment by UmasouTTT** [comment]
By the way, for the currently supported tags, please refer to the tag_dict file in the Parameter folder.

**Comment by OKtang** [comment]
can I get your wechat number, I've had a problem for many days. my wechat: abc12369tt
